components/layout/CarouselBuilder.tsx

```diff
@@ -0,0 +1,49 @@
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+import { Grid } from '@mui/material';
+import React from 'react';
```

Comment:
is this import necessary? I don't see React being used

Reply:
We are using the airbnb eslint config and one of the rules is that you have to import React on all pages with jsx or get a lint error. We could theoretically add the rule to the set that it ignores?